### PR TITLE
github: Add labels, colors

### PR DESCRIPTION
**Pull request by YakDriver**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Adds labels to workflow that were previously not captured. Aligns label colors with HashiCorp's color schemes for applications.
* https://www.hashicorp.com/brand

Output from acceptance testing:

```
NA
```

**Update:** Since submitting this PR, HashiCorp has changed colors slightly. The colors listed in this comment are not accurate. They are correct in the PR itself.

Uses these groupings:

`623ce4` - Terraform Purple
service
provider

`491dd7` - Darker Terraform Purple
partition/aws-us-gov
partition/aws-iso-b
partition/aws-cn
partition/aws-iso

`ca2171` - Consul Pink
needs-triage

`962fab`
enhancement
new-resource
new-data-source

`f04e54` - Boundary Red
bug
crash
breaking-change
regression

`ef4349` - Darker Boundary Red
waiting-response

`00bc7f` - Nomad Green
tests
terraform-0.15
terraform-0.14
terraform-0.11
terraform-0.12
alpha-testing

`b0288e`
documentation
technical-debt
proposal
thinking
question
linter

`14c6cb` - Waypoint Turquoise
size/XS
size/XL
size/L
size/M
size/S
size/XXL

`1563ff` - HashiCorp (& Vagrant) Blue
upstream-terraform
upstream
go
dependencies

`00acff` - Packer Blue
good first issue
help wanted
examples

`828a90`
stale
hashibot/ignore
new
windows
reinvent
hacktoberfest
github_actions

### CODE REVIEW

File: infrastructure/repository/labels-workflow.tf [hunk 1]
```diff
@@ -1,15 +1,50 @@
 variable "workflow_labels" {
```
Comment (anGie44) [inline infrastructure/repository/labels-workflow.tf:1]: my only concern is that some of the more urgent labels (bug/regression/crash) seem to blend in quite easily with the less urgent (documentation/needs-triage) so glancing over the issue list requires more 👀 ...or could just be my screen brightness 😅 i do feel though that "proposal" should get its own color as its more rare/lives in a different category imo to "documentation"

File: infrastructure/repository/labels-workflow.tf [hunk 2]
```diff
@@ -1,15 +1,50 @@
...
+    "provider"                       = "623ce4",
+    "needs-triage"                   = "ca2171",
+    "enhancement"                    = "962fab",
+    "new-resource"                   = "962fab",
+    "new-data-source"                = "962fab",
+    "bug"                            = "f04e54",
+    "crash"                          = "f04e54",
+    "breaking-change"                = "f04e54",
+    "regression"                     = "f04e54",
+    "waiting-response"               = "ef4349",
```
Comment (anGie44) [inline infrastructure/repository/labels-workflow.tf:12]: these 2 look very similar 😄 

Screen Shot 2021-02-25 at 10 19 19 AM

**Comment by YakDriver** [comment]
Some label organizing...

Label | Category | Urgency | Uses | % Issues | Cur Color
-- | -- | -- | -- | -- | --
service/* (201 labels) | Identity | 3-Low | 3443 | 105.2% | 623CE4
needs-triage | Classify | 2-Medium | 1345 | 41.1% | CA2171
enhancement | Classify | 3-Low | 929 | 28.4% | 962FAB
size/* (6 labels) | Size | 3-Low | 629 | 19.2% | 14C6CB
bug | Classify | 1-High | 607 | 18.6% | F04E54
tests | Testing | 3-Low | 515 | 15.7% | 00BC7F
documentation | Classify | 3-Low | 440 | 13.4% | B0288E
provider | Identity | 3-Low | 301 | 9.2% | 623CE4
new-resource | Classify | 3-Low | 196 | 6.0% | 962FAB
technical-debt | Classify | 3-Low | 127 | 3.9% | B0288E
new-data-source | Classify | 3-Low | 74 | 2.3% | 962FAB
partition/* (4 labels) | Identity | 3-Low | 59 | 1.8% | 491DD7
upstream-terraform | External | 3-Low | 48 | 1.5% | 1563FF
proposal | Classify | 3-Low | 40 | 1.2% | B0288E
upstream | External | 3-Low | 39 | 1.2% | 1563FF
linter | Internal | 3-Low | 36 | 1.1% | B0288E
question | Community | 3-Low | 33 | 1.0% | B0288E
good first issue | Community | 2-Medium | 26 | 0.8% | 00ACFF
stale | Community | 3-Low | 26 | 0.8% | 828A90
crash | Alert | 1-High | 21 | 0.6% | F04E54
thinking | Internal | 3-Low | 20 | 0.6% | B0288E
breaking-change | Alert | 1-High | 18 | 0.6% | F04E54
terraform-* (4 labels) | Testing | 3-Low | 17 | 0.5% | 00BC7F
waiting-response | Community | 2-Medium | 14 | 0.4% | EF4349
dependencies | External | 3-Low | 9 | 0.3% | 1563FF
examples | Community | 3-Low | 9 | 0.3% | 00ACFF
prerelease-tf-testing | Testing | 3-Low | 9 | 0.3% | 00BC7F
go | External | 3-Low | 5 | 0.2% | 1563FF
help wanted | Community | 3-Low | 5 | 0.2% | 00ACFF
hashibot/ignore | Unused | 3-Low | 4 | 0.1% | 828A90
regression | Alert | 1-High | 4 | 0.1% | F04E54
new | Unused | 3-Low | 3 | 0.1% | 828A90
windows | Unused | 3-Low | 1 | 0.0% | 828A90
github_actions | Internal | 3-Low | 0 | 0.0% | 828A90
hacktoberfest | Community | 3-Low | 0 | 0.0% | 828A90
pr/no-changelog | Internal | 3-Low | 0 | 0.0% | 828A90
reinvent | Community | 3-Low | 0 | 0.0% | 828A90

**Review by anGie44 — APPROVED** [review]
@YakDriver i think let's get this one so we have the labels correctly tracked? (modifications can be addressed later if they're not working for us; afict no hay ningún problema 👍) ..also looks like just a go mod check failure to address atm

**Pull request merged**